src/Assets/LightSkyDefense/Scripts/Credit.cs

```diff
@@ -7,8 +7,14 @@
 [RequireComponent(typeof(Interactable))]
 public class Credit : MonoBehaviour
 {
-    public const int value = 5;
-        
+    public int Value = 5;
+    private AudioSource source;
+    public AudioClip pickupSound;
+    void Awake()
```

Comment:
Can we add a whitespace between methods and properties

```suggestion

    void Awake()
```